domains/imide.json

```diff
@@ -5,6 +5,12 @@
         "discord":  "555915168932954112"
     },
     "record": {
-        "CNAME": "imide-is-a-dev.pages.dev"
+        "CNAME": "imide-is-a-dev.pages.dev",
```

Comment:
You can't use CNAME with other records

also for discord, you need to make a `_discord.imide` domain

Reply:
the docs do not provide any info regarding mail. what is the standard/recommended way to handle that?